### PR TITLE
[feature/1022] Added Handler to move assets as part of workflow task

### PR DESCRIPTION
**Pull request by kanika1901kapoor**
added ProjectTaskAssetMoverHandler.java - handler to move assets that caters to all types of configurable tasks, 
updated ReviewTaskAssetMoverHandler.java to expose shared logic

### CODE REVIEW

**Comment by kanika1901kapoor** [comment]
[project-task-asset-mover-overlay.zip](https://github.com/Adobe-Consulting-Services/acs-aem-commons/files/1330496/project-task-asset-mover-overlay.zip)

The overlay will add the 3 fields-On Approve Move To, On Reject Move To, Review Conflict Resolution - to any task workflow step as shown below :
image

**Comment by kanika1901kapoor** [comment]
@davidjgonzalez - any review comments on this one?

File: bundle/src/main/java/com/adobe/acs/commons/dam/impl/ProjectTaskAssetMoverHandler.java [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                name = EventConstants.EVENT_TOPIC,
+                propertyPrivate = true
+        ),
+
+        /* Event filters support LDAP filter syntax and have access to event.getProperty(..) values */
+        /* LDAP Query syntax: https://goo.gl/MCX2or */
+        @Property(
+                label = "Event Filters",
+                // Only listen on events associated with nodes that end with /jcr:content
+                value = "(&(TaskTypeName=dam:review)(EventType=TASK_COMPLETED))",
```
Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/dam/impl/ProjectTaskAssetMoverHandler.java]: Im not super familiar with Projects; is dam:review the appropriate event task type for Projects?

File: bundle/src/main/java/com/adobe/acs/commons/dam/impl/ProjectTaskAssetMoverHandler.java [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+                description = "Event Filters used to further restrict this event handler; Uses LDAP expression against event properties. Defaults to: (&(TaskTypeName=dam:review)(EventType=TASK_COMPLETED))",
+                name = EventConstants.EVENT_FILTER,
+                propertyPrivate = false
+        )
+})
+@Service
+public class ProjectTaskAssetMoverHandler extends ReviewTaskAssetMoverHandler {
+    
+    private static final String DEFAULT_APPROVED_TERM = "approved";
```
Comment (davidjgonzalez) [inline bundle/src/main/java/com/adobe/acs/commons/dam/impl/ProjectTaskAssetMoverHandler.java]: For my own edification; what is the significance of this value? Will this simply show up for display? Can it be anything without impact to anything in projects (for example, Asset Review Tasks has very specific values required)

**Pull request closed without merging**